understanding/error-identification.html

```diff
@@ -55,161 +53,132 @@
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.3.1:</span>Error Identification (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.3.1:</span>エラーの特定 (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>If an <a href="#dfn-input-error">input error</a> is automatically detected, the item that is in error is identified and the error
-                     is described to the user in text.
+                  <p><a href="#dfn-input-error">入力エラー</a>が自動的に検出された場合は、エラーとなっている箇所が特定され、そのエラーが利用者にテキストで説明される。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Users know an error exists and what is wrong.</dd>
+                  <dt>目標</dt>
+                  <dd>利用者は、エラーが存在し、何が間違っているかを知ることができる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide descriptive notification of errors.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>エラーの説明的な通知を提供する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Flagging errors helps people with reduced sight and cognitive disabilities resolve
-                     them.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>エラーにフラグを立てると、視力の低下及び認知障害のある人がエラーを解決するのに役立つ。
```

Comment:
「フラグを立てる」はもちろん合ってはいるのですが、この表現で通じますかね……?
もう少し和語的な表現、たとえば「目印をつける」「目立たせる」「はっきりさせる」などを検討しても良いようにも思いますが、どうでしょうか。悩ましいところではありますが。

```suggestion
                  <dd>エラーを目立たせることは、視力の低下及び認知障害のある人がエラーを解決するのに役立つ。
```